crates/api_crud/src/user/create.rs

```diff
@@ -149,6 +149,7 @@ impl PerformCrud for Register {
       password_encrypted: Some(data.password.to_string()),
       show_nsfw: Some(data.show_nsfw),
       email_verified: Some(false),
+      discussion_languages: Some(LanguageIdentifier::all_languages()),
```

Comment:
This is the `und` tag?

Reply:
No, new users start with all languages enabled. So they will see content in all languages, and have to change the setting to disable languages they dont understand. I will add a param for this, so lemmy-ui can initialize it based on browser language.